linux_basics_session02/steps/step05.md

```diff
@@ -36,10 +36,9 @@ cd
 <Quiz id="q1" choices={[ { valid: false, value: "/shared/data/nr"}, 
 						 { valid: false, value: "test"}, 
 						 { valid: false, value: "your HOME directory"}, 
```

Comment:
This quiz had a duplicate answer. Also, when I ran the steps, the `pwd` I got was `/shared/data` and not `nr`, let me know if that looks right to you